crates/forge/bin/cmd/script/cmd.rs

```diff
@@ -83,14 +85,13 @@ impl ScriptArgs {
         let mut decoder = self.decode_traces(&script_config, &mut result, &known_contracts)?;
 
         if self.debug {
-            return self.run_debugger(
-                &decoder,
+            let debugger = DebuggerArgs {
+                debug: result.debug.clone().unwrap_or(vec![]),
```

Comment:
`unwrap_or_else` might be more performant here

Reply:
`unwrap_or_default` even better, wdyt ?

Reply:
yep sounds good